### PR TITLE
[CST] - 72065 Change 'Submitted on' to 'Received on'

### PR DESCRIPTION
**Pull request by pmclaren19**
## Summary

**Updated multiple files so that the language was 'Received on' instead of 'Submitted on':**
- src/applications/claims-status/components/ClaimDetailLayout.jsx
- src/applications/claims-status/components/ClaimsListItem.jsx
- src/applications/claims-status/components/appeals-v2/ClaimsListItemV2.jsx
- src/applications/claims-status/components/ClaimsListItemV3.jsx
- src/applications/claims-status/components/StemClaimListItem.jsx
- src/applications/claims-status/components/StemClaimListItemV3.jsx
- src/applications/claims-status/components/appeals-v2/AppealListItemV2.jsx
- src/applications/claims-status/components/appeals-v2/AppealListItemV3.jsx

**Updated the following evss files so that the language was 'Received on' instead of 'Submitted on':**
-src/applications/claims-status/components/evss/ClaimDetailLayout.jsx

**Updated multiple test files so that the language was 'Received on' instead of 'Submitted on':**
- src/applications/claims-status/tests/components/ClaimDetailLayout.unit.spec.jsx
- src/applications/claims-status/tests/components/ClaimListItem.unit.spec.jsx
- src/applications/claims-status/tests/components/appeals-v2/ClaimListItemV2.unit.spec.jsx
- src/applications/claims-status/tests/components/ClaimListItemV3.unit.spec.jsx
- src/applications/claims-status/tests/components/StemClaimListItem.unit.spec.jsx
- src/applications/claims-status/tests/components/StemClaimListItemV3.unit.spec.jsx
- src/applications/claims-status/tests/components/appeals-v2/AppealListItemV2.unit.spec.jsx
- src/applications/claims-status/tests/components/appeals-v2/AppealListItemV3.unit.spec.jsx

**Updated the following evss test files so that the language was 'Received on' instead of 'Submitted on':**
- src/applications/claims-status/tests/components/evss/ClaimDetailLayout.unit.spec.jsx

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#72065

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Evss  | Screenshot 2024-01-22 at 11 49 03 AM | Screenshot 2024-01-22 at 3 09 45 PM |
| Lighthouse | Screenshot 2024-01-22 at 11 49 21 AM | Screenshot 2024-01-22 at 11 53 12 AM|
| Evss  | Screenshot 2024-01-22 at 2 56 29 PM | Screenshot 2024-01-22 at 11 47 47 AM |
| Lighthouse  | Screenshot 2024-01-22 at 2 58 45 PM | Screenshot 2024-01-22 at 11 47 30 AM |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Tasks

- [x]  Update the V2 claim card
- [x]  Update the V3 claim card
- [x]  Update the subheading on the EVSS version of the claim details page
- [x]  Update the subheading on the Lighthouse version of the claim details page
- [x]  Update any tests that are associated with the components mentioned above

## Acceptance criteria

- [x] Claims cards are updated to use 'Received on' language
- [x]  Claim details subheading is updated to use 'Received on' language
- [ ]  Update https://github.com/department-of-veterans-affairs/va.gov-team/issues/72065 so sitewide team is aware of this change
- [ ]  Notify VA Mobile, VA Chatbot team, and other status tool teams about the language changes and why we made those changes.

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/claims-status/components/ClaimDetailLayout.jsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default function ClaimDetailLayout(props) {
```
Comment (pmclaren19) [inline src/applications/claims-status/components/ClaimDetailLayout.jsx:65]: Lighthouse claim details page

File: src/applications/claims-status/components/evss/ClaimDetailLayout.jsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default function ClaimDetailLayout(props) {
   } else if (claim !== null) {
     const claimTitle = `Your ${claimType} claim`;
     const formattedClaimDate = formatDate(claim.attributes.dateFiled);
-    const claimSubheader = `Submitted on ${formattedClaimDate}`;
+    const claimSubheader = `Received on ${formattedClaimDate}`;
```
Comment (pmclaren19) [inline src/applications/claims-status/components/evss/ClaimDetailLayout.jsx:62]: EVSS claim details page

File: src/applications/claims-status/components/ClaimsListItemV3.jsx [hunk 3]
```diff
@@ -72,7 +72,7 @@ export default function ClaimsListItemV3({ claim }) {
     <ClaimCard
       title={getTitle(claim)}
       label={inProgress ? 'In Progress' : null}
-      subtitle={`Submitted on ${formattedReceiptDate}`}
+      subtitle={`Received on ${formattedReceiptDate}`}
```
Comment (pmclaren19) [inline src/applications/claims-status/components/ClaimsListItemV3.jsx:75]: Lighthouse claim details card

File: src/applications/claims-status/components/ClaimsListItem.jsx [hunk 4]
```diff
@@ -93,7 +93,7 @@ export default function ClaimsListItem({ claim }) {
       </ul>
       <div className="card-status">
         <p>
-          <strong>Submitted on:</strong> {formattedReceiptDate}
+          <strong>Received on:</strong> {formattedReceiptDate}
```
Comment (pmclaren19) [inline src/applications/claims-status/components/ClaimsListItem.jsx:96]: Evss claim card.

File: src/applications/claims-status/components/appeals-v2/ClaimsListItemV2.jsx [hunk 5]
```diff
@@ -63,7 +63,7 @@ export default function ClaimsListItem({ claim }) {
       </ul>
       <div className="card-status">
         <p>
-          <strong>Submitted on:</strong> {formattedReceiptDate}
+          <strong>Received on:</strong> {formattedReceiptDate}
```
Comment (pmclaren19) [inline src/applications/claims-status/components/appeals-v2/ClaimsListItemV2.jsx:66]: EVSS claim details card.

File: src/applications/claims-status/components/StemClaimListItemV3.jsx [hunk 6]
```diff
@@ -37,7 +37,7 @@ export default function StemClaimListItemV3({ claim }) {
   return (
     <ClaimCard
       title="Edith Nourse Rogers STEM Scholarship application"
-      subtitle={`Submitted on ${formattedReceiptDate}`}
+      subtitle={`Received on ${formattedReceiptDate}`}
```
Comment (pmclaren19) [inline src/applications/claims-status/components/StemClaimListItemV3.jsx:40]: This claim details card is used when the following is true:

claim type = 'education_benefits_claims'
useLighthouse = true
useNewClaimCards is disabled.

File: src/applications/claims-status/components/appeals-v2/AppealListItemV2.jsx [hunk 7]
```diff
@@ -106,7 +106,7 @@ export default function AppealListItem({ appeal, name, external = false }) {
       {requestEvent && (
         <div className="card-status">
           <p>
-            <strong>Submitted on:</strong>{' '}
+            <strong>Received on:</strong>{' '}
```
Comment (pmclaren19) [inline src/applications/claims-status/components/appeals-v2/AppealListItemV2.jsx:109]: This claim details card is used when the following is true:
- claim type is a part of the appeal types that were passed in to YourClaimsPageV2.jsx
- useLighthouse = false
- useNewClaimCards is disabled.

File: src/applications/claims-status/components/appeals-v2/AppealListItemV3.jsx [hunk 8]
```diff
@@ -83,7 +83,7 @@ export default function AppealListItemV3({ appeal, name }) {
       title={appealTitle}
       subtitle={
         requestEvent &&
-        `Submitted on ${moment(requestEvent.date).format('MMMM D, YYYY')}`
+        `Received on ${moment(requestEvent.date).format('MMMM D, YYYY')}`
```
Comment (pmclaren19) [inline src/applications/claims-status/components/appeals-v2/AppealListItemV3.jsx:86]: This claim details card is used when the following is true:
- claim type is a part of the appealtypes that were passed in to YourClaimsPageV2.jsx
- useLighthouse = true
- useNewClaimCards is enabled.

**Review by jerekshoe — APPROVED** [review]

**Pull request merged**